pyscf/lib/cmake.arch.inc

```diff
@@ -0,0 +1,26 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -ffast-math")
```

Comment:
cmake.arch.inc should not be in the repo. Only examples of it...

Reply:
So, what do you think to add lines for turning off MPI for new users?